pageserver/src/tenant/storage_layer/inmemory_layer.rs

```diff
@@ -32,10 +32,14 @@ use super::{
     ValuesReconstructState,
 };
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy, Hash)]
+pub(crate) struct InMemoryLayerKey(page_cache::FileId);
```

Comment:
Why is it called Key and not InMemoryLayerFileId ?

Key is confusing because InMemoryLayer already is a map of some sort of key to value.